config/packages/twig.yaml

```
@@ -1,5 +1,7 @@
 twig:
     default_path: '%kernel.project_dir%/templates'
+    form_themes:
+        - "bootstrap_5_layout.html.twig"
```

```suggestion
        - 'bootstrap_5_layout.html.twig'
```
Using simple quotes to be consistent with the previous line